tickets/src/test/java/no/unit/nva/publication/ticket/create/CreateTicketHandlerTest.java

```diff
@@ -91,6 +91,22 @@ void shouldPersistTicketWhenPublicationExistsUserIsOwnerAndPublicationMeetsTicke
         assertThatLocationHeaderHasExpectedFormat(publication, location);
         assertThatLocationHeaderPointsToCreatedTicket(location);
     }
+
+    @Test
+    void shouldBePossibleToCreateDoiTicketForPublicationWithPublishedMetadataOnly()
```

Comment:
Can this not be a parameterised test (likely an existing test?) to make it obvious that both of these statuses are valid

Reply:
fikset